src/components/new-safe/create/logic/index.test.ts

```diff
@@ -301,7 +298,7 @@ describe('create/logic', () => {
           chainBuilder()
             .with({ chainId: '137' })
             // Multichain creation is toggled on
-            .with({ features: [FEATURES.SAFE_141, FEATURES.COUNTERFACTUAL, FEATURES.MULTI_CHAIN_SAFE_CREATION] as any })
+            .with({ features: [FEATURES.COUNTERFACTUAL, FEATURES.MULTI_CHAIN_SAFE_CREATION] as any })
```

Comment:
Nit: as before regarding mocking the `recommendedMasterCopyVersion` as `'1.4.1'`.